tools/remark-preset-lint-node/index.js

```diff
@@ -0,0 +1,44 @@
+// @see https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md
+
+'use strict';
+
+module.exports.plugins = [
+  require('remark-lint'),
+  require('remark-lint-checkbox-content-indent'),
+  require('remark-lint-definition-spacing'),
+  require('remark-lint-fenced-code-flag'),
+  require('remark-lint-final-definition'),
+  require('remark-lint-final-newline'),
+  require('remark-lint-hard-break-spaces'),
```

Comment:
Are trailing spaces (necessary for this rule) possible in our docs? Should they be eliminated but properly configured git on committing?